manifests/init.pp

```diff
@@ -254,6 +254,12 @@
 
   if $enable_docker {
     include pulpcore::plugin::container
+    include foreman_proxy_content::container
+    if !$shared_with_foreman_vhost {
```

Comment:
This also works:
```suggestion
    unless $shared_with_foreman_vhost {
```

However, i do think it should also make `include foreman_proxy_content::container` dependent on it.

Reply:
I've updated and re-tested.